srml/staking/src/lib.rs

```diff
@@ -16,7 +16,157 @@
 // You should have received a copy of the GNU General Public License
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
-//! Staking manager: Periodically determines the best set of validators.
+//! # Staking Module
+//! 
+//! <!-- Original author of paragraph: @gavofyork -->
+//! The staking module is the means by which a set of network maintainers (known as "authorities" in some contexts and "validators" in others) are chosen based upon those who voluntarily place funds under deposit. Under deposit, those funds are rewarded under normal operation but are held at pain of "slash" (expropriation) should they be found not to bee discharging their duties properly. 
```

Comment:
"Under deposit, those funds are rewarded" - Are the _funds_ rewarded, or the _maintainer_? I think it should either say that the staked pot increases or the maintainer is rewarded, but rewarding funds is strange.

Reply:
let's settle for fund owner? though maintainer is also clear.

Reply:
I actually think this is clear enough. This is the very frst section and using less keywords makes mroe sense. Funds are placed. Funds are rewarded. poeriod. 

Of course, if anyone would read the full doc they would never confuse where the rewards are going.

In fact, funds can end of rewarding either the maintainer, or the fund intself (being being forwarded to the stash account directly).

Reply:
How about, "those funds' owners are rewarded..."

This actually makes it clear that,
1. The funds can have multiple owners (the maintainer plus the nominators).
2. It is the beneficiaries who are being rewarded, not the funds.

Reply:
> Funds are placed. Funds are rewarded. poeriod.

It doesn't make sense to reward an inanimate object. To me, a reward implies gratification of some sort, and "funds" can't feel gratification, but their owners are happy to receive the additional funds.